### PR TITLE
Add ubuntu 20.04 instructions

### PR DESCRIPTION
**Pull request by XertroV**
Took a bit to figure out (hopefully helps someone), and dependencies were a problem since xbuild/msbuild don't fetch them. Simple solution, just no easy to find.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,16 +22,29 @@ Nibriboard is a product of an initial idea that I thought I could get done in ab
...
+
+Windows users may need to use a _Visual Studio Command Prompt_ if the `msbuild` command isn't in your `PATH` environment variable.
 
 Note that if you're intending to use Nibriboard over the internet or an untrusted network, you should proxy it behind nginx to provide TLS, as Nirbiboard doesn't handle HTTPS on it's own.
 
+### Getting Started with Ubuntu 20.04 (and probably others)
+
+This may or may not work with Ubuntu's provided mono packages, though was tested with those provided by mono-project. Note: if for whatever reason you don't have `mono-roslyn` (which provides `msbuild`), use `mono-xbuild` and `xbuild` in their respective places.
+
+* packages: `sudo apt install mono-devel mono-roslyn libnewtonsoft-json-cil-dev`
```
Comment (sbrl) [inline README.md:42]: I thought that newtonsoft.json was fetched from Nuget?

Comment (XertroV) [reply README.md:42]: You're welcome :)

> I thought that newtonsoft.json was fetched from Nuget?

I _think_ so too, but I never confirmed it. The JSON errors were the first ones I googled, and it turned out there was a package, so I installed that, and found out how to do nuget properly later. I'll alter it and make a note that if there are any issues the the JSON libs to try installing that.

**Comment by XertroV** [comment]
I agree to release this and any future work contributed to sbrl/Nibriboard under the Mozilla Public License 2.0.

**Comment by sbrl** [comment]
Thanks again :-)

Hopefully at some point I'll have some more time to come back to this project. It's such a cool idea, and I've put so much effort into it already it seems a shame to waste it

**Pull request merged**